### PR TITLE
#479: Provide more verbose exception message when AJAX fails.

### PR DESCRIPTION
**Pull request by marcoscano**
Very rough, just as a proof-of-concept, but this seems to help a lot while troubleshooting where a given error is coming from.

### CODE REVIEW

File: src/Drupal/DrupalExtension/Context/MinkContext.php [hunk 1]
```diff
@@ -98,7 +98,7 @@ public function beforeJavascriptStep($event)
         }
         $text = $event->getStep()->getText();
         if (preg_match('/(follow|press|click|submit)/i', $text)) {
-            $this->iWaitForAjaxToFinish();
+            $this->iWaitForAjaxToFinish($event);
```
Comment (jhedstrom) [inline src/Drupal/DrupalExtension/Context/MinkContext.php]: This variable doesn't appear to be defined anywhere...

Comment (Berdir) [reply src/Drupal/DrupalExtension/Context/MinkContext.php]: `public function beforeJavascriptStep($event)`

It is passed to the method :)

Comment (jhedstrom) [reply src/Drupal/DrupalExtension/Context/MinkContext.php]: haha, oops.

**Comment by jhedstrom** [comment]
Great, thanks!

**Pull request merged**